derive_builder/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 - customize setter names via `#[builder(setter(name="..."))]`
+- customize build_fn name via `#[builder(build_fn(name="..."))]`
+- suppress build method generation via `#[builder(build_fn(skip))]`
+- perform pre-build validation via `#[builder(build_fn(validator="path::to::fn"))]`
```

Comment:
Setter/getter are well known concepts and everyone knows they are methods. If we go with `build_fn` I think `validate_fn` is the name that is most consistent with that and should be used.